favorites/views.py

```diff
@@ -1,9 +1,19 @@
 from django.shortcuts import render
 from django.views.generic.list import ListView
 from .models import Favorite
+from django.views.generic.edit import CreateView
+from .forms import NewFavoriteForm
 
 class FavoritesView(ListView):
     model = Favorite
     template_name = "favorites/favorites.html"    
     
 favorites = FavoritesView.as_view()
+
+class NewFavoriteView(CreateView):
+    model = Favorite
+    fields = ['name']
+    template_name = "favorites/new-favorite.html"
+    success_url = '../'
```

Comment:
You're specifying a url here, so this is basically the same as `http://localhost:8000/../` which doesn't totally make sense, but is effectively the same as `http://localhost:8000/`. But I would suggest you just redirect to `/`.